### PR TITLE
bugfix/tg-3963-e2e-test-search-results-kadastraal-subject

### PR DESCRIPTION
**Pull request by jasperswart**

### CODE REVIEW

**Comment by jasperswart** [comment]
test this please

File: e2e/helpers/search.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+/**
+ *
+ * @description this function enters a search query and then submits the form
+ */
+module.exports = function (query) {
+    browser.driver.findElement(by.css('input.c-search-form__input')).sendKeys(query);
```
Comment (RRMoelker) [inline e2e/helpers/search.js]: qa plz

**Comment by jasperswart** [comment]
test this please

**Comment by jasperswart** [comment]
test this please

File: e2e/search-results-kadastraal-subject.test.js [hunk 2]
```diff
@@ -19,24 +19,48 @@ describe('Zoekresultaten kadastraal subjecten', () => {
...
+                expect(searchResults.categories(4).headerCount).not.toBe('0');
+                expect(searchResults.categories(4).listCount).not.toBe(0);
 
-                    expect(searchResults.categories(3).header).toContain('Maatschappelijke activiteiten');
+                headerCountEmployeePlus = searchResults.categories(4).headerCount;
 
-                    expect(searchResults.categories(4).header).toContain('Kadastrale subjecten');
-                    expect(searchResults.categories(4).listCount).not.toBe(0);
-                });
+                expect(dp.isLargerThan(headerCountEmployee, headerCountEmployeePlus)).toEqual(true);
```
Comment (RRMoelker) [inline e2e/search-results-kadastraal-subject.test.js]: https://stackoverflow.com/a/32475300/2583290

**Pull request merged**